deployment/pom.xml

```diff
@@ -33,6 +33,10 @@
             <groupId>io.quarkiverse.githubaction</groupId>
             <artifactId>quarkus-github-action</artifactId>
         </dependency>
+        <dependency>
+          <groupId>io.quarkus</groupId>
+          <artifactId>quarkus-devtools-common</artifactId>
+        </dependency>
```

Comment:
I'm puzzled you have to adjust the original extension?

Reply:
I don't. Left overs after moving things around. Removing it.